src/content/app/tools/blast/views/blast-form/BlastForm.tsx

```diff
@@ -60,12 +60,23 @@ const BlastForm = () => {
 
 const Main = () => {
   const isSmallViewport = useMediaQuery('(max-width: 1900px)');
+  const [sequenceValidityStatus, setSequenceValidityStatus] = useState({});
```

Comment:
"status" (singular) suggests that there is only one thing; whereas in your code it is a key-value collection of things, where the key is an index, and the value is a boolean representing whether this sequence is valid or not.